core/math/a_star.cpp

```diff
@@ -425,8 +425,8 @@ void AStar::_bind_methods() {
 
 	ClassDB::bind_method(D_METHOD("clear"), &AStar::clear);
 
-	ClassDB::bind_method(D_METHOD("get_closest_point", "to_pos"), &AStar::get_closest_point);
-	ClassDB::bind_method(D_METHOD("get_closest_pos_in_segment", "to_pos"), &AStar::get_closest_pos_in_segment);
+	ClassDB::bind_method(D_METHOD("get_closest_point", "to_position"), &AStar::get_closest_point);
+	ClassDB::bind_method(D_METHOD("get_closest_pos_in_segment", "to_position"), &AStar::get_closest_pos_in_segment);
```

Comment:
Should be `get_closest_pos*ition*_in_segment` I guess.

Reply:
Yeah I noticed that when I did the cpp methods. The rename for this one is in the "method" commit. I left it there for review but I'll do a fixup of both commits before the merge.